server/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rent-check-server",
+  "version": "1.0.0",
+  "description": "",
+  "author": "Michael Watts <hello@michaelwatts.co>",
+  "keywords": [],
+  "license": "ISC",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon ./bin/app",
```

Comment:
I think we need to add the nodemon to the devDependencies just in case someone not installing it globally